From 6245131c0c328d2ed876d87cc20b176e2f24a769 Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Tue, 9 Aug 2016 15:12:06 -0700
Subject: [PATCH] of: Export of_device_uevent_modalias for DRM_MIPI_DSI.

In order to make DSI panel modules able to load automatically, we need
to emit the OF modalias uevent so that their MODULE_DEVICE_TABLE(of,
...) can match against it.

Signed-off-by: Eric Anholt <eric@anholt.net>
---
 drivers/of/device.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/of/device.c
+++ b/drivers/of/device.c
@@ -287,3 +287,4 @@ int of_device_uevent_modalias(struct dev
 
 	return 0;
 }
+EXPORT_SYMBOL(of_device_uevent_modalias);
